config/environments/development.rb

```diff
@@ -38,4 +38,20 @@
 
   # Raises error for missing translations
   # config.action_view.raise_on_missing_translations = true
+  config.action_mailer.default_url_options = { host: 'localhost:3000' }
+
+  config.action_mailer.delivery_method = :smtp
+
+  config.action_mailer.perform_deliveries = true
+
+  config.action_mailer.smtp_settings = {
+      address: 'smtp.gmail.com',
+      port: 587,
+      domain: ENV['GMAIL_DOMAIN'],
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.